README.rst

```diff
@@ -28,6 +28,7 @@ Requirements
 ============
 This branch requires 3.1.3 version MU1 or later of the HPE 3PAR firmware
 This branch requires 4.0.0 version of the HPE Primera firmware
```

Comment:
Change the version to 4.3.1 instead of 4.0.0

Reply:
Addressed review comments